templates/zerver/help/change-your-org-name.md

```diff
@@ -0,0 +1,18 @@
+# Change your organization's name
+Only an administrator can change the name of the organization.
+
+Follow the following steps to change the name of the organization using an administrator account:
+* Click on the cog (![cog](/static/images/help/cog.png)) icon in the upper right corner
```

Comment:
Yep, numbers will be better since these steps should be performed in a sequence.